.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2026-02-16 - IDOR and Wildcard Enumeration in Server Actions
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)